### ISSUE TITLE
Adding icon for GitLab

### ISSUE THREAD

**Issue by gokatz** [empress/field-guide#9]
This would a nice UI tweak. It would be great if we could support GitLab icons for Gitlab URLS

**Comment by mansona** [comment]
Thanks for the issue @gokatz can you explain more about what you are looking for? 🤔  I don't fully understand what you mean by "GitLab icons for GitLab URLS"

**Comment by gokatz** [comment]
Oops. sorry for the missing context. We've added a [configuration for adding GitHub URL](https://github.com/empress/field-guide#configuring) of the repo. Can we add support for adding GitLab URL as well? If we do so, we can display the GitLab icon for that case here:

<img width="311" alt="Screenshot 2020-03-09 at 5 23 24 PM" src="https://user-images.githubusercontent.com/15158776/76210998-48a35180-622b-11ea-9e9d-f30285131d0b.png">

**Comment by mansona** [comment]
ha ha ha! yes that's probably a good idea 😂 

I probably need to have a more generic way to implement this than just a `github: 'https://github.com/blah` config 🤔 Any thoughts on the design? on first thoughts, it should probably be some sort of object that would allow arbitrary numbers of social (like maybe you had a twitter **and** a github or something)

what are your thoughts on something like: 

```
'field-guide': {
  name: 'Product Name',
  copyright: 'This is the default copyright string - update before publishing',
  social: {
    github: 'https://github.com/empress/field-guide?update-with-your-repo-url',
    twitter: 'https://twitter.com/emberjs'
  }
}
```

And as for icons, I could support a common set of them (github, gitlab, twitter etc.) but if you had something strange there you would just need to provide an icon file with the same name in `public/images/icons/somethingStrange.svg`.

What do you think?

**Comment by gokatz** [comment]
You've pretty much designed the API virtually, @mansona 😄 
`social` hash sounds cool. Maybe can we add support for "Strange" things at a later stage upon requests as I'm not sure what will I fill up there?

**Comment by gokatz** [comment]
Hi @mansona Can I take this up?

**Comment by mansona** [comment]
@gokatz yes absolutely! I'm happy to give you some guidance to get it working if you would like 👍 I'm not too sure how discoverable this is right now 🤔

**Comment by gokatz** [comment]
Sure. Will have a look and let you know
